samples/cpp/benchmark_app/main.cpp

```diff
@@ -555,19 +570,16 @@ int main(int argc, char* argv[]) {
         // ----------------- 8. Querying optimal runtime parameters
         // -----------------------------------------------------
         next_step();
-        // output of the actual settings that the device selected based on the hint
-        if (!ov_perf_hint.empty()) {
-            for (const auto& device : devices) {
-                std::vector<std::string> supported_config_keys =
-                    core.get_metric(device, METRIC_KEY(SUPPORTED_CONFIG_KEYS));
-                slog::info << "Device: " << device << slog::endl;
-                for (const auto& cfg : supported_config_keys) {
-                    try {
-                        slog::info << "  {" << cfg << " , " << compiledModel.get_config(cfg).as<std::string>();
-                    } catch (...) {
-                    };
+        // output of the actual settings that the device selected
+        for (const auto& device : devices) {
+            std::vector<std::string> supported_config_keys = core.get_metric(device, METRIC_KEY(SUPPORTED_CONFIG_KEYS));
+            slog::info << "Device: " << device << slog::endl;
+            for (const auto& cfg : supported_config_keys) {
+                try {
+                    slog::info << "  {" << cfg << " , " << compiledModel.get_config(cfg).as<std::string>();
                     slog::info << " }" << slog::endl;
-                }
```

Comment:
if some config value is nit supported (e.g. by the AUTO) there is no closing bracket and line break
the following line should be duplicated in the catch statement:
_slog::info << " }" << slog::endl;_

now the output for AUTO is :

_[Step 8/11] Setting optimal runtime parameters
[ INFO ] Device: AUTO
[ INFO ]   {PERFORMANCE_HINT , LATENCY }
[ INFO ]   {PERFORMANCE_HINT_NUM_REQUESTS ,   {MULTI_DEVICE_PRIORITIES , CPU,GPU }
[ INFO ]   {MULTI_WORK_MODE_AS_AUTO ,   {PERF_COUNT , NO }
[ INFO ]   {EXCLUSIVE_ASYNC_REQUESTS ,   {MODEL_PRIORITY ,   {ALLOW_AUTO_BATCHING , [Step 9/11] Creating infer requests and preparing input blobs with data_